update-motd.d/30-pengwin-setup

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+printf " Pengwin include 'Pengwin Setup', allows new Pengwin users to get started on Linux immediately"
+printf "\n"
+printf " To execute run: pengwin-setup"
+printf "\n"
+printf " To check for updates run: pengwin-setup update"
```

Comment:
To check for updates, run: pengwin-setup update

Reply:
Done